packages/runtime/runtime-definitions/package.json

```
@@ -48,8 +58,12 @@
 	"types": "lib/public.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "flub generate entrypoints --outFileLegacyBeta legacy --outDir ./dist",
-		"api-extractor:esnext": "flub generate entrypoints --outFileLegacyBeta legacy --outDir ./lib --node10TypeCompat",
+		"api-extractor:commonjs": "concurrently \"npm:api-extractor:commonjs:*\"",
+		"api-extractor:esnext": "concurrently \"npm:api-extractor:esnext:*\"",
+		"api-extractor:commonjs:legacyBeta": "flub generate entrypoints --outFileLegacyBeta legacy --outDir ./dist",
+		"api-extractor:esnext:legacyBeta": "flub generate entrypoints --outFileLegacyBeta legacy --outDir ./lib --node10TypeCompat",
+		"api-extractor:commonjs:legacyAlpha": "flub generate entrypoints --outFileLegacyAlpha legacyAlpha --outDir ./dist",
+		"api-extractor:esnext:legacyAlpha": "flub generate entrypoints --outFileLegacyAlpha legacyAlpha --outDir ./lib --node10TypeCompat",
```

You should be able to combine all of this into:

```suggestion
		"api-extractor:commonjs": "flub generate entrypoints --outFileLegacyBeta legacy --outFileLegacyAlpha legacyAlpha --outDir ./dist",
		"api-extractor:esnext": "flub generate entrypoints --outFileLegacyBeta legacy --outFileLegacyAlpha legacyAlpha --outDir ./lib --node10TypeCompat",
```